apps/docs/docs/quickstart.mdx

```diff
@@ -4,23 +4,37 @@ import { InstallTabs } from '@site/src/components/InstallTabs';
 
 # Quickstart
 
+Setup is easy. 
+
+Once you understand how to use `ts-rest`, developer experience is like butter.  🧈
+
 ## Installation
 
 Install the core package
 
 <InstallTabs packageName="@ts-rest/core" />
 
+
 ## Create a contract
 
 <Tabs groupId="contract">
   <TabItem value="zod" label="Zod">
-  <InstallTabs packageName="zod" />
-<p>Create a contract, this should ideally be shared between your consumers and producers, e.g. in a shared library in a monorepo, or a shared npm package.</p>
 
-<p>
-  In this case we're using Zod, this has some nice features, like enabling body
-  parsing and OpenAPI type generation.
-</p>
+1. **Install Zod** <InstallTabs packageName="zod" />
```

Comment:
The bullet points are creating some alignment issues. Maybe it should be better to use headings?

```suggestion
### Install Zod
<InstallTabs packageName="zod" />
```